docs/database/server/server.md

```diff
@@ -84,6 +84,44 @@ post_id string INDEX FK >- Post.id
 size number
 width number
 
+Group
+-
+id PK string INDEX # server-generated 
```

Comment:
Error - server-database-v1
Line 89: The field cannot be a primary key and an index, since a primary key is an index